backend/coreapp/views/compiler.py

```diff
@@ -14,10 +16,42 @@
 boot_time = now()
 
 
-def endpoint_updated(request: Request) -> datetime:
+def endpoint_updated(request: Request, **_: typing.Any) -> datetime:
     return max(Preset.most_recent_updated(request), boot_time)
 
 
+class SingleCompilerDetail(APIView):
+    @condition(last_modified_func=endpoint_updated)
+    def get(
```

Comment:
Since this function is just returning a list of compilers, I think the last_modified time would be the application startup time. I we have other example funcs that use that?